.travis.yml

```diff
@@ -33,6 +36,14 @@ script:
  - cabal test --show-details=always
  - cabal sdist
  - cabal haddock | grep "100%" | wc -l | grep "4"
+ - if [[ "$TRAVIS_OS_NAME" != "osx" ]]; then docker build .; fi
```

Comment:
@javcasas would be good to run a command inside the container as part of the building process, so we check the binary is working properly.

```
docker build -t hapistrano
docker run hapistrano --version
```